opnode/test/system_test.go

```diff
@@ -26,206 +25,135 @@ import (
-	const bssHDPath = "m/44'/60'/0'/0/4"
-
-	// System Config
-	cfg := &systemConfig{
-		mnemonic: "squirrel green gallery layer logic title habit chase clog actress language enrich body plate fun pledge gap abuse mansion define either blast alien witness",
-		l1: gethConfig{
-			nodeConfig: &node.Config{
-				Name:   "l1geth",
-				WSHost: "127.0.0.1",
-				WSPort: 9090,
-			},
-			ethConfig: &ethconfig.Config{
-				NetworkId: 900,
-			},
-		},
-		l2Verifier: gethConfig{
-			nodeConfig: &node.Config{
-				Name:   "l2gethVerify",
-				WSHost: "127.0.0.1",
-				WSPort: 9091,
-			},
-			ethConfig: &ethconfig.Config{
-				NetworkId: 901,
-			},
-		},
-		l2Sequencer: gethConfig{
-			nodeConfig: &node.Config{
-				Name:   "l2gethSeq",
-				WSHost: "127.0.0.1",
-				WSPort: 9092,
-			},
-			ethConfig: &ethconfig.Config{
-				NetworkId: 901,
-			},
-		},
-		premine: map[string]int{
-			"m/44'/60'/0'/0/0": 10000000,
-			"m/44'/60'/0'/0/1": 10000000,
-			"m/44'/60'/0'/0/2": 10000000,
+func defaultSystemConfig(t *testing.T) SystemConfig {
+	return SystemConfig{
+		Mnemonic: "squirrel green gallery layer logic title habit chase clog actress language enrich body plate fun pledge gap abuse mansion define either blast alien witness",
+		Premine: map[string]int{
+			cliqueSignerHDPath: 10000000,
+			transactorHDPath:   10000000,
 			l2OutputHDPath:     10000000,
 			bssHDPath:          10000000,
 		},
-		cliqueSigners:          []string{"m/44'/60'/0'/0/0"},
-		depositContractAddress: derive.DepositContractAddr.Hex(),
-		l1InfoPredeployAddress: derive.L1InfoPredeployAddr.Hex(),
-	}
-	// Create genesis & assign it to ethconfigs
-	initializeGenesis(cfg)
-
-	// Start L1
-	l1Node, l1Backend, err := l1Geth(cfg)
-	require.Nil(t, err)
-	defer l1Node.Close()
-
-	err = l1Node.Start()
-	require.Nil(t, err)
-
-	err = l1Backend.StartMining(1)
-	require.Nil(t, err)
-
-	l1Client, err := ethclient.Dial(endpoint(cfg.l1.nodeConfig))
-	require.Nil(t, err)
-	l1GenesisID, _ := getGenesisInfo(l1Client)
-
-	// Start L2
-	l2Node, _, err := l2Geth(cfg)
-	require.Nil(t, err)
-	defer l2Node.Close()
-
-	err = l2Node.Start()
-	require.Nil(t, err)
-
-	l2Client, err := ethclient.Dial(endpoint(cfg.l2Verifier.nodeConfig))
-	require.Nil(t, err)
-	l2GenesisID, l2GenesisTime := getGenesisInfo(l2Client)
-
-	// Start L2
-	l2SequencerNode, _, err := l2SequencerGeth(cfg)
-	require.Nil(t, err)
-	defer l2SequencerNode.Close()
-
-	err = l2SequencerNode.Start()
-	require.Nil(t, err)
-
-	l2SequencerClient, err := ethclient.Dial(endpoint(cfg.l2Sequencer.nodeConfig))
-	require.Nil(t, err)
-
-	// BSS
-	bssPrivKey, err := cfg.wallet.PrivateKey(accounts.Account{
-		URL: accounts.URL{
-			Path: bssHDPath,
+		BatchSubmitterHDPath:       bssHDPath,
+		CliqueSignerDerivationPath: cliqueSignerHDPath,
+		DepositContractAddress:     common.HexToAddress("0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001"),
+		L1InfoPredeployAddress:     common.HexToAddress("0x4242424242424242424242424242424242424242"),
```

Comment:
This address is incorrect and it should be importing the constant that is defined in the codebase